_locales/fr/messages.json

```diff
@@ -19,40 +19,40 @@
         "message": "Soumission invalide"
     },
     "429": {
-        "message": "Vous avez soummetez trop de endossements, il y a vraiment cette montant?"
+        "message": "Vous cherchez à envoyer beaucoup de segments, il y en a vraiment autant ?"
     },
     "409": {
-        "message": "Déjas soummis"
+        "message": "Déja soumis"
     },
     "502": {
-        "message": "Le serveur ne fonctionne pas. Message le développeur."
+        "message": "Le serveur ne fonctionne pas. Contactez le développeur."
     },
     "channelWhitelisted": {
-        "message": "Cette channel est sur la liste blanche!"
+        "message": "Cette chaine est sur la liste blanche !"
     },
     "Sponsor": {
-        "message": "endossement"
+        "message": "message commercial"
     },
     "Sponsors": {
-        "message": "endossements"
+        "message": "messages commerciaux"
     },
     "Segment": {
-        "message": "section d'endossement"
+        "message": "segment de message commercial"
     },
     "Segments": {
-        "message": "section d'endossements"
+        "message": "segments de message commercial"
     },
     "noticeTitle": {
-        "message": "Endossement Passer"
+        "message": "Passer ce message"
```

Comment:
This should be past-tense, as the message was just skipped.

Reply:
"Message passé" ("Sponsor skipped")